packages/qmp/qmp.0.20.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "xen-api@lists.xen.org"
+authors: [ "Dave Scott" ]
+homepage: "https://github.com/xapi-project/ocaml-qmp"
+bug-reports: "https://github.com/xapi-project/ocaml-qmp/issues"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
Should only the tests be marked as unavailable on 32bit architecture or the package as a whole?
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test & arch != "arm32" & arch != "x86_32"}
```
```
#=== ERROR while compiling qmp.0.20.0 =========================================#
# context              2.2.0~alpha3~dev | linux/x86_32 | ocaml-base-compiler.4.14.1 | pinned(https://github.com/xapi-project/ocaml-qmp/releases/download/v0.20.0/qmp-0.20.0.tbz)
# path                 ~/.opam/4.14/.opam-switch/build/qmp.0.20.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p qmp -j 127
# exit-code            1
# env-file             ~/.opam/log/qmp-7-3a4a19.env
# output-file          ~/.opam/log/qmp-7-3a4a19.out
### output ###
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -g -bin-annot -I lib_test/.messages.eobjs/byte -I /home/opam/.opam/4.14/lib/ounit2 -I /home/opam/.opam/4.14/lib/ounit2/advanced -I /home/opam/.opam/4.14/lib/seq -I /home/opam/.opam/4.14/lib/stdlib-shims -I /home/opam/.opam/4.14/lib/yojson -I lib/.libqmp.objs/byte -no-alias-deps -o lib_test/.messages.eobjs/byte/messages.cmo -c -impl lib_test/messages.ml)
# File "lib_test/messages.ml", line 24, characters 56-66:
# 24 |   "event-powerdown.json",          Event { timestamp = (1258551470, 802380); event = "POWERDOWN"; data = None };
#                                                              ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
```

Reply:
Seems like the package as a whole, this looks like a deserialization issue